### PR TITLE
Readability/Grammar improvements + Adding more pros/cons for some patterns

### PR DESCRIPTION
**Pull request by 123gkc**
- Improved readability and grammar for a few patterns.
- Added more pros and cons for the patterns based on my personal experience in scaling ML services

### CODE REVIEW

File: Operation-patterns/Model-in-image-pattern/design_en.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ A difficulty of the pattern is that the image building latency tends to get long
 
 ## Cons
 - You need to define a complete pipeline of a model training and image building.
+- Not scalable if you have to host multiple models in your prediction service. For example, if you have a separate model for feature extraction, or an ensemble of models for prediction etc, then the memory requirements for your service are very high.
```
Comment (shibuiwilliam) [inline Operation-patterns/Model-in-image-pattern/design_en.md:20]: [Question]
This point can be worked through with dividing the images by models. i.e., making an image for feature extraction, another for a model and so forth, as described in the prep-pred pattern and horizontal microservice pattrern.

**Review by shibuiwilliam — COMMENTED** [review]
Thanks!
I have one comment.
Other than that, LGTM!

File: Operation-patterns/Model-in-image-pattern/design_en.md (hunk 1 above)
Comment (shibuiwilliam) [reply Operation-patterns/Model-in-image-pattern/design_en.md:20]: @123gkc

Comment (123gkc) [reply Operation-patterns/Model-in-image-pattern/design_en.md:20]: Makes sense. Understood now.

**Comment by 123gkc** [comment]
Incorporated the suggestions

**Review by shibuiwilliam — APPROVED** [review]
lgtm!

**Pull request merged**